src/Aspire.ProjectTemplates.9.0.net9/templates/aspire-apphost/Aspire.AppHost1.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <Sdk Name="Aspire.AppHost.Sdk" Version="!!REPLACE_WITH_LATEST_VERSION!!" />
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
I think these templates should just target `net9.0` and not rely on the parameter replacement configuration?

Reply:
Agreed, the net9.0 tempaltes should target net9.0 natively, and the template.json should be updated with the proper replaces to essentially make it a noop

Reply:
done.